### PR TITLE
Add a WIP charter for SIG multicluster

### PR DESCRIPTION
**Pull request by pmorie**
Initial stab at a WIP charter for SIG multicluster (mostly cribbed from README)

### CODE REVIEW

File: sig-multicluster/charter.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# SIG Multicluster Charter
+
+This charter is a WIP.
+
+SIG Multicluster focuses on solving common challenges related to:
+
+- Managing/administering multiple Kubernetes clusters
```
Comment (irfanurrehman) [inline sig-multicluster/charter.md]: Probably rewording needed.
`Managing/administering centralised information about multiple kubernetes clusters` ?? or something else.
The prior (`cluster administration/management` as the term) seems to suggest stuff `cluster-lifecycle` folks might be involved with.
I understand that the same wording is currently used on the sig-multicluster page; IMHO needs correction.

Comment (pmorie) [reply sig-multicluster/charter.md]: I think there's probably an overlap here. As I understand it, cluster-lifecycle is about creating new clusters. As I understand it, the wording currently in the README pertains to administering multiple clusters from the same source of truth. I'm fine striking it for now if people would like, but I expect this to be in scope for SIG multicluster.

Comment (irfanurrehman) [reply sig-multicluster/charter.md]: Seems reasonable, lets go ahead with what this PR has.

**Comment by timothysc** [comment]
@pmorie This needs a bit of work to follow the updated sig charter.  

https://github.com/kubernetes/community/tree/master/committee-steering/governance

Also requires scope boundaries.

**Comment by michelleN** [comment]
@pmorie - Thanks for your time on this! We added a [charter template](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md) in case that helps you a bit more. Feel free to reach out if you need clarification on anything. 

We're merging all charters with non-blocking items on Wednesday in case you want to close the loop on this but no pressure.

Cheers

**Comment by philips** [comment]
Thank you for your patience as we work to your SIG charter merged.

Through the process of reviewing the [first 11 charters][charters] the Steering Committee (SC) found a lot of copy/paste language caused by our poorly designed initial charter template. To fix this there is a new [SIG charter template][template] which focuses on the main things we want to see in charters: scope and responsibilities.

Please consider updating to this charter template and putting focus into the scope and responsibilities. SC members will focus more on a deep review of scope more than anything else and using this template will help with that focus.

[charters]: https://docs.google.com/spreadsheets/d/1i_8oM1KlmXo90zIIgNTdOg-M6xesTggFLnTkgmscMlI/edit#gid=0
[template]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md

**Pull request closed without merging**